blueflood-udp/src/main/java/com/rackspacecloud/blueflood/service/udp/MainIngestor.java

```diff
@@ -100,7 +100,7 @@ private static void checkConfiguration() {
 
         log.info("Configuration is good");
 
-        boolean useZookeeper = !"NONE".equals(config.getStringProperty(CoreConfig.ZOOKEEPER_CLUSTER));
+        boolean useZookeeper = false;
         log.info("Using zookeeper? " + useZookeeper);
```

Comment:
can we delete the boolean and this log statement?  I think that's the only use of the variable.